src/localization/bundles/en.json

```diff
@@ -478,6 +480,15 @@
     "deploy.get.log.error": "Failed to get the log for playbook {0} - {1}. {2}",
     "deploy.progress.failure": "Cloud provisioning has failed. Please review the errors in the log, hit Back, address the issue by editing any necessary files, re-run Validate, and Deploy again.",
 
+    "update.progress.heading": "Cloud update deployment in progress",
+    "update.prepare.progress.heading": "Preparing model update deployment in progress",
+    "update.progress.failure": "Failed to deploy cloud update",
+    "update.deploy.confirm": "The deployment of the model update may disrupt your cloud.  Do you want to proceed?",
```

Comment:
I think the EOS guidelines discourage "you" and "your" ... @dotNomad may be able to confirm... I'd lean towards something like `Deploying a model update may disrupt cloud operations. Proceed with model update?`

Reply:
The guide doesn't explicitly limit "you" and "your" but it does have rules around it. Such as
> Show our expertise by being precise and polite but not personal or overly friendly.